src/server/auth/cmds/cmds.go

```diff
@@ -242,9 +245,11 @@ func DeactivateCmd(ctx context.Context, pachctlCfg *pachctl.Config) *cobra.Comma
 	var enterprise bool
 	deactivate := &cobra.Command{
 		Short: "Delete all ACLs, tokens, admins, IDP integrations and OIDC clients, and deactivate Pachyderm auth",
-		Long: "Deactivate Pachyderm's auth and identity systems, which will delete ALL auth " +
+		Long: "This command deactivates Pachyderm's auth and identity systems, which will delete ALL auth " +
 			"tokens, ACLs and admins, IDP integrations and OIDC clients, and expose all data " +
 			"in the cluster to any user with cluster access. Use with caution.",
```

Comment:
I don't think this is very clear.  I'd say that turning off exposes data to everyone on the network.